dev/com.ibm.ws.org.jboss.resteasy.resteasy.core/bnd.bnd

```diff
@@ -37,7 +37,11 @@ Export-Package: \
   org.jboss.resteasy.util;version=4.5.2.Final, \
   org.jboss.resteasy.util.snapshot;version=4.5.2.Final
 
+# Using version=! in order to not have a version attached to the import for packages that were removed
+# from Java after Java 8.  Doing this keeps the import like before Java 11 support. It will get the
+# packages from Java when using Java 8 or earlier and from the new shipped bundles for Java 9 and later.
 Import-Package: \
+  javax.activation;version=!, \
```

Comment:
Should javax.annotation and maybe javax.annotation.security also be added?

Reply:
This bundle doesn't require it.
Do I need to add them anyway "just in case"?

Reply:
Why do you say it doesn't require it?  It is in the Imports in the manifest file when I look at it.

Reply:
When I start a server, the listed imports were the only issues. I resolved these and I started a server cleanly with Java 8. I suppose if it's in the manifest file I can add the ones you listed as well, but just starting a server isn't causing any issues with my proposed change.

Reply:
When processing annotations if they are not found, they are just ignored and do not produce an error, so you would not see an issue starting the server for missing annotation packages.

Reply:
updated